sdk/resourcemanager/Azure.ResourceManager.Core/samples/Sample1_HelloWorld.md

```diff
@@ -0,0 +1,29 @@
+# Example: Getting a susbcription
+
+>Note: Before getting started with the samples, make sure to go trought [prerequisites guide](<--./docs/Prerequistes.md-->).
+
+The following code shows how to get the default subscription:
+
+```csharp Snippet:HelloWorld_GetDefaultSubscription
+ArmClient armClient = new ArmClient(new DefaultAzureCredential());
+Subscription subscription = armClient.DefaultSubscription;
+Console.WriteLine(subscription.Id);
```

Comment:
since we are giving two examples of the same thing lets use the same console.writeline in both